feat: setup advanced filter to have default options updated packages

tnoelcke opened the pull request:
### Updates Made
I've updated the advanced filter so that it is possible to have a default selected button.

### Contributing developer checklist
- [x] I've updated source branch with the latest changes from dev.
- [ ] I've added/changed unit tests for components with functionality.
- [ ] I've added/updated storybook for any component that I've changed.
- [ ] I've reviewed each jsdoc header in regards to the changes that I've made and updated them.

### Reviewing developer checklist
- [ ] I've pulled this branch to my local machine.
- [ ] I've inspected the changes on storybook.
- [ ] I've run the test suite and all tests have passed

danvanhorn commented:
Did you update the tests? These changes would have probably changed our storybook snapshots.

tnoelcke commented:
I had an issue with a new version of react router breaking how the docs get built. Reverted back to an older version of react router as I had updated it.
